### PR TITLE
docs(analytics): update analytics docs

### PR DESCRIPTION
**Pull request by danielpickett**
Update docs for analytics component to be clearer about required data attributes and to more prominently link to the logging tutorial.

### CODE REVIEW

File: docusaurus/docs/components/analytics/analytics.md [hunk 1]
```diff
@@ -2,42 +2,67 @@
 title: <Analytics />
 ---
 
-Component allowing page and click events to be tracked.
+This Analytics component is part of Availity's toolset for tracking 'click', 'blur', 'focus' and page load events in your application. For a helpful overview of how to setup these tools, read our [Setting Up Logging](https://availity.github.io/availity-workflow/recipes/logging/) guide. Logging in a React application at Availity usually involves imports from this React focused [@availity/analytics](https://www.npmjs.com/package/@availity/analytics) package and from [@availity/analytics-core](https://www.npmjs.com/package/@availity/analytics-core). Starting with the Setting Up Logging guide will give you a better understanding of how they work together.
+
+You may also find it helpful read [these docs on the AvAnalytics class](https://availity.github.io/sdk-js/resources/analytics), since the Analytics component implements an instance of AvAnalytics, and most of the props are forwarded directly to it.
+
+If you only need Splunk reports, use avLogMessagesApi. If you're working with Availity's DMA team for reporting through Insights, use avLogMessagesApiV2
```
Comment (chrishavekost) [inline docusaurus/docs/components/analytics/analytics.md]: > If you only need Splunk reports, use avLogMessagesApi.

Let's remove this line. `avLogMessageApiV2` can log to both Splunk and DMA. If users need to log to DMA for Insights reports, they need to pass a `spaceId` or `configurationId` in their payload

**Review by chrishavekost — DISMISSED** [review]

File: docusaurus/docs/components/analytics/analytics.md [hunk 2]
```diff
@@ -2,42 +2,67 @@
 title: <Analytics />
 ---
 
-Component allowing page and click events to be tracked.
+This Analytics component is part of Availity's toolset for tracking 'click', 'blur', 'focus' and page load events in your application. For a helpful overview of how to setup these tools, read our [Setting Up Logging](https://availity.github.io/availity-workflow/recipes/logging/) guide. Logging in a React application at Availity usually involves imports from this React focused [@availity/analytics](https://www.npmjs.com/package/@availity/analytics) package and from [@availity/analytics-core](https://www.npmjs.com/package/@availity/analytics-core). Starting with the Setting Up Logging guide will give you a better understanding of how they work together.
+
+You may also find it helpful read [these docs on the AvAnalytics class](https://availity.github.io/sdk-js/resources/analytics), since the Analytics component implements an instance of AvAnalytics, and most of the props are forwarded directly to it.
```
Comment (gregmartDOTin) [inline docusaurus/docs/components/analytics/analytics.md]: Missing a to inbetween helpful and read

File: docusaurus/docs/components/analytics/analytics.md (hunk 1 above)
Comment (danielpickett) [reply docusaurus/docs/components/analytics/analytics.md]: Oh snap. Yes, we talked about removing. My bad.

Comment (danielpickett) [reply docusaurus/docs/components/analytics/analytics.md]: FYI - I'm also adding some more detail on plugins and logging to the docs for analytics-core in sdk-js.

Comment (gregmartDOTin) [inline docusaurus/docs/components/analytics/analytics.md]: This is a subjective comment, so feel free to disregard.

I would change to `... use the avLogMessagesApi plugin.` and `... use the avLogMessagesApiV2 plugin.` If I hadn't read the Docs linked above yet, I wouldn't know what avLogMessagesApi and avLogMesagessApiV2 were.

File: docusaurus/docs/components/analytics/analytics.md [hunk 3]
```diff
@@ -53,7 +78,7 @@ When an event is created, if `true`, will traverse up the DOM tree from the give
 
 #### `attributePrefix?: string`
 
-Prefix to apply on event attributes you want to stripe off components on click. **Default:** `data-analytics`
+Prefix to apply on event attributes you want to strip off components on click. **Default:** `data-analytics`
```
Comment (gregmartDOTin) [inline docusaurus/docs/components/analytics/analytics.md]: Do we need more detailed documentation on what the attributePrefix does? Seemed like there was confusion there.

File: docusaurus/docs/components/analytics/analytics.md (hunk 1 above)
Comment (gregmartDOTin) [reply docusaurus/docs/components/analytics/analytics.md]: Ah, I see your convo with Chris now. Ignore me lol

File: docusaurus/docs/components/analytics/analytics.md (hunk 3 above)
Comment (danielpickett) [reply docusaurus/docs/components/analytics/analytics.md]: Sure, I could provide some example code here.

Also, I could not for the life of me figure out how to get `eventModifiers` below to do anything useful. If you have any insights on that, do tell!

**Review by jordan-a-young — APPROVED** [review]

**Review by gregmartDOTin — APPROVED** [review]

**Pull request merged**